components/partials/home/HomeModes.vue

```diff
@@ -54,16 +54,6 @@
             </i18n>
           </div>
         </div>
-        <div class="px-4 lg:p-8">
-          <h4 class="uppercase text-2xl pt-8 pb-4 font-medium">
-            {{ $t('homepage.modes.spa.title') }}
```

Comment:
Did you also removed it from i18n @debs-obrien ?

Reply:
now removed. thanks. forgot all about removing it from that file.